beacon-chain/sync/batch_verifier.go

```diff
@@ -83,17 +84,31 @@ func verifyBatch(verifierBatch []*signatureVerifier) {
 		return
 	}
 	aggSet := verifierBatch[0].set
-	verificationErr := error(nil)
 
 	for i := 1; i < len(verifierBatch); i++ {
 		aggSet = aggSet.Join(verifierBatch[i].set)
 	}
-	verified, err := aggSet.Verify()
-	switch {
-	case err != nil:
-		verificationErr = err
-	case !verified:
-		verificationErr = errors.New("batch signature verification failed")
+	verificationErr := error(nil)
```

Comment:
first time ive seen this instead of `var verificationErr error`

Reply:
It might be more idiomatic to declare it like that, will change it